CMSgov/cmcs-eregulations
# [Tech Debt] Upgrade Storybook to latest

**Description**

Interactively upgrades Storybook from 6.4.9 to 6.5.12 using the suggested `npx sb@latest upgrade` command.

**This pull request changes**

- Upgrades storybook and its related libraries
- Adds `.npmrc` config file, created by Storybook's interactive installer, to enable `legacy-peer-deps`
- Adds `eslint-plugin-storybook` but does not configure it properly (yet)

**Steps to manually verify this change**

1. See if Storybook deploys successfully below as part of the PR experimental deployment process
2. run `make storybook` locally to see if it runs Storybook as expected on your local machine
